### DISCUSSION TITLE
Connecting to unix socket?

### DISCUSSION THREAD

**Discussion by nepthar** [jasync-sql/jasync-sql#262] [Q&A] [answered]
Hello, I'm attempting to use jasync on google cloud. They've got a system which allows you to securely connect to your cloud SQL instance via a unix socket. I cannot, however, seem to get jasync to connect over a socket. Is this possible at the moment?

**Comment by andy-yx-chen** [comment, marked as answer]
There was a PR https://github.com/jasync-sql/jasync-sql/pull/176 but I did not have time to add test cases for it, maybe you can continue from there

**Reply by nepthar** [reply]
Thanks, I'll take a look
